gallery/tutorials/tutorials/image_expansion.py

```diff
@@ -28,8 +28,10 @@
 # -------------------
 
 # Load the images from NumPy array, 10 images of 70S Ribosome with size 129 x 129
-DATA_DIR = "data"
-org_images = np.load(os.path.join(DATA_DIR, "example_data_np_array.npy")).T
+file_path = os.path.join(
+    os.path.dirname(os.getcwd()), "data", "example_data_np_array.npy"
+)
+org_images = np.load(file_path).T
```

Comment:
Do we know where this transpose comes from? If not, may want to make an issue to track this down.

Reply:
Just from not wanting to repack the numpy data (binary).  Eventually we can get rid of it. These other tutorials need updating (but not today).

```
>>> a = np.load('example_data_np_array.npy')
>>> a.shape
(129, 129, 10)
```

Reply:
Ah got it. Maybe put a comment noting that the array is stored in the “wrong order”?

Reply:
Sure, added.